projects/plugins/boost/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
 	"extends": "@tsconfig/svelte/tsconfig.json",
-	"include": [ "app/assets/src/js/**/*" ],
+	"include": [ "app/assets/src/js/**/*", "../../js-packages/components/**/*" ],
```

Comment:
This is needed for rollup build which fails when importing TypeScript components from `js-packages`. The reason for that being that `@rollup/plugin-typescript` by default only handles files inside the project directory, thus files from outside are excluded.

Reply:
Seems like there should be a better way to do it than breaking the package boundary in the tsconfig.json.

Reply:
`@rollup/plugin-typescript` doesn't work as expected. I even tried to create a separate `tsconfig-rollup.json` and passed it to the plugin but it fails. It looks like `@rollup/plugin-typescript` is buggy.

May be Boost team can help?

Reply:
I have created an issue (#23961) and added a possible solution as well.